JSONable.java

```diff
@@ -0,0 +1,6 @@
+package org.json;
+
+public interface JSONable {
+	public JSONObject toJSONObject();
+	public String toJSON();
```

Comment:
What is the point of the `toJSON()` interface method? It seems redundant to the [`JSONString#toJSONString()`](https://github.com/stleary/JSON-java/blob/master/JSONString.java#L17) method

Reply:
I think only in
object.toJSON() instead object.toJSONObject().toString()

Reply:
I would remove it. It's not obvious that that was the intent and I don't really see it as that helpful if there is no default implementation (like in java8 which we don't support yet)